Makefile

```diff
@@ -197,6 +197,11 @@ endif
 # On Windows only build tsh. On all other platforms build teleport, tctl,
 # and tsh.
 BINS_default = teleport tctl tsh tbot
+ifneq ($(CHECK_RUST),)
+ifneq ($(CHECK_CARGO),)
+BINS_default += gravitational-ssh-fdpass
```

Comment:
@espadolini nit, most teleport customers don't know what gravitational means, so adding this in the binary name is going to be confusing, let's drop `gravitational`?

Reply:
I've been told to avoid "teleport" and "tbot" as prefixes for the binary name to avoid problems in our build scripts, and I feel like it would be rude to drop a binary in the path that users really don't need to interact with directly (this would be used by the autogenerated ssh_config) and that's specific to our product without at least some hint that it's for our product.

Perhaps "fdpass-teleport-ssh"?

Reply:
"helper-teleport-ssh-fdpass"?

Reply:
@espadolini `fdpass-teleport` is OK, afaik this does not do anything specific, and it will be clear to folks looking at the binary that it's part of teleport